<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QWidget" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QStackedWidget" name="widgets">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="lessonTableWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="lessonViewTable">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="teachingLoadWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="teacherNameView">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>The teacher you clicked will be shown here.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="goUpButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Go Up</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../resources/main.qrc">
           <normaloff>:/root/dialogs/go-parent.png</normaloff>:/root/dialogs/go-parent.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="teachingLoadView">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select the kind of view by which you see the lessons:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="selectViewComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>By a complete list</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>By teacher's name</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>selectViewComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CreditView</receiver>
   <slot>resetView(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>688</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goUpButton</sender>
   <signal>clicked()</signal>
   <receiver>CreditView</receiver>
   <slot>goUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teachingLoadView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>CreditView</receiver>
   <slot>goDeeper(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetView(int)</slot>
  <slot>goUp()</slot>
  <slot>goDeeper(QModelIndex)</slot>
 </slots>
</ui>
